src/api/app/jobs/send_event_emails_job.rb

```diff
@@ -26,12 +26,31 @@ def perform
 
   def create_rss_notifications(event)
     event.subscriptions.each do |subscription|
-      Notification::RssFeedItem.create(
+      notification_params = {
         subscriber: subscription.subscriber,
         event_type: event.eventtype,
         event_payload: event.payload,
         subscription_receiver_role: subscription.receiver_role
-      )
+      }
+      Notification::RssFeedItem.create(notification_params.merge!(notification_dynamic_params(event)))
+    end
+  end
+
+  def notification_dynamic_params(event)
+    case event.eventtype
+    when 'ReviewWanted', 'RequestCreate', 'RequestStatechange'
+      {
+        notifiable_type: 'BsRequest',
+        notification_id: event.payload['id'],
+        bs_request_oldstate: (event.payload['oldstate'] if event.eventtype == 'RequestStatechange')
```

Comment:
BTW I also think you need to say state, otherwise you will not be able to tell which state it switched to **at the time** of the Notification. Only at which state it is **now**...

Reply:
Ops, you're right. Pull request is in the way.